config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller:latest
+      - image: docker.io/jont828/cluster-api-azure-controller-arm64:e2e
```

Comment:
Is it a miss -- The 'jont828' part of the image?

Reply:
Oops, let me fix that!